packages/components/src/count/test/index.jsx

```diff
@@ -2,6 +2,7 @@
  * @jest-environment jsdom
  */
 import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Hey @jeyip curious to know what this line does? :)

Reply:
Thanks for the question -- I almost forgot about this line change.

After cloning the tests into @automattic/components, the expectation helpers `.toHaveTextContent` and `.toHaveClass` weren't available in the package testing environment. 

It might be better to move this to some jest config file as a follow-up, but I wasn't sure if it was premature because it's only used for this one file in @automattic/components